src/tox_travis/after.py

```diff
@@ -81,10 +80,10 @@ def travis_after():
     print('All required jobs were successful.')
 
 
-def after_config_matches():
+def after_config_matches(config):
     """Determine if this job should wait for the others."""
-    config = py.iniconfig.IniConfig('tox.ini')
-    section = config.sections.get('travis:after', {})
+    tox_ini = get_iniconfig(config)
+    section = tox_ini.sections.get('travis:after', {})
```

Comment:
But then I suppose I'd want to name it to `get_ini`. Either way works. Pick whichever color you like better.